be/src/olap/rowset/segment_v2/bitmap_index_writer.h

```diff
@@ -0,0 +1,51 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+
+#include "common/status.h"
+#include "gen_cpp/segment_v2.pb.h"
+#include "gutil/macros.h"
+
+namespace doris {
+
+class TypeInfo;
+class WritableFile;
+
+namespace segment_v2 {
+
+class BitmapIndexWriter {
+public:
+    static Status create(const TypeInfo* typeinfo, std::unique_ptr<BitmapIndexWriter>* res);
+
+    BitmapIndexWriter() = default;
+    virtual ~BitmapIndexWriter() = default;
+
+    virtual void add_values(const void* values, size_t count) = 0;
```

Comment:
void is enough? or change it to Status later?

Reply:
I can't think of any error condition for now. And sure we can change it to Status later when it's necessary.